java/mykola-melnyk/src/com/zoolatech/lecture1/tasks/_1/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.zoolatech.lecture1.tasks._1;
```

Comment:
please remove all files that are not related to the HW #4

Reply:
Can you please specify which files exactly do I need to remove?

Reply:
all files from the folder `java/mykola-melnyk/src/com/zoolatech/lecture3
![Screenshot 2022-08-22 at 17 03 58](https://user-images.githubusercontent.com/108929952/185940604-8761cafb-26d6-4a9f-95f3-1d84e8e8a317.png)

Reply:
Deleted all the contents of the lecture 3 folder. But I must have misunderstood something. What about the lectures 1&2? Do I have to remove the contents of thhat folders too? I don't get the idea, sorry. Can you please explain the logic behind this? Thank you :)

Reply:
When you create a new branch from the current branch, the new branch will contain all commits from the current one. I think when you created the branch for the current MR ("mykola-melnyk-java-4"), you did it from the branch with your 3rd homework ("mykola-melnyk-java-3"). So, now the branch for this MR contains all changes from both homeworks. To prevent such issues you need to create new branches for your homeworks from the master branch only.

The simplest way to resolve this issue might be to create a new branch from the master branch, use [git cherry pick](https://git-scm.com/docs/git-cherry-pick) tool to copy commits from the old branch, and remove the old branch.

To simplify the process you can keep the files from other homeworks.

Reply:
Thank you so much for your detailed answer!